api/childSubmissions/childSubmissionsModel.js

```diff
@@ -20,9 +20,21 @@ async function updateSubmissionBySubmissionId(id, changes) {
     .returning('*');
 }
 
+//rename to addSubmissionPage
+async function addS3Submission(submission) {
+  return db('submissionPages').insert(submission).returning('*');
+  // const [newS3Img] = await db('s3').insert(imgUrl).returning('*');
```

Comment:
Do you need this commented out code?

Reply:
no, thanks for the heads up. It was removed